### ISSUE TITLE
UPDATE 20/07/21

### ISSUE THREAD

**Issue by KlementineJBS** [OpenSourceTB/OSTB_Series4_OpenCyclams#3]
## Lockdown blues 

Unfortunately, Sydney (where this project is based) has been plunged into a lockdown that's impeded the current experimental work. With a little luck, things will get moving again in a few weeks' time, but for the moment most lines of work are rather at a standstill. 

## What's been achieved so far? 

We don't have any finished compounds as yet, but are working towards the click-coupled bis-adamantyl open cyclams with the successful synthesis of one coupling partner, the [1-azido-adamantane](https://github.com/KlementineJBS/ELN_USYD_Honours/wiki/Azidoadamantane-synthesis).

<p align="center">
<img width="500" src="https://user-images.githubusercontent.com/78768991/126249865-3fce84d5-286c-44a6-b604-6df278b405e1.jpg">
</p>

## What's still cooking?

We're working on the [bis-allylation of the same 1,6-diaminohexane scaffold](https://github.com/KlementineJBS/ELN_USYD_Honours/wiki/Allylation-of-1%2C6-diaminohexane) as part of our DOS strategy.  This experiment has failed or produced very low yields in the past, so we're in the middle of optimisation for it, looking mostly at solvents and purification methods. Hopefully the results of this [current experiment](https://github.com/KlementineJBS/ELN_USYD_Honours/issues/19) will shed some light on how to improve it. 

<p align="center">
<img width="500" src="https://user-images.githubusercontent.com/78768991/115826617-89215700-a44e-11eb-995e-d2d5baa1d73f.jpg">
</p>

Similarly, the first step of our proposed route to the "open cyclam" requires a bis-propargylation of the desired polyamine scaffold. Currently, we're working on [propargylating the 1,6-diaminohexane scaffold](https://github.com/KlementineJBS/ELN_USYD_Honours/wiki/Propargylation-of-1%2C6-diaminohexane). The main issue we seem to be facing is the production of the mono-propargylated product, which complicates isolation of our desired compound.  

<p align="center">
<img width="500" src="https://user-images.githubusercontent.com/78768991/121116730-57c3e580-c85a-11eb-8ede-9a8e955506c1.png">
</p>

Finally, we're working on the other hydrophobic pendant groups that will be coupled to our "open cyclams"., but the azidomethylferrocene has given us some trouble in the [final step](https://github.com/KlementineJBS/ELN_USYD_Honours/wiki/Azidomethylferrocene-synthesis), and has not yet been successfully isolated or characterised. 

![Actual azidomethylferrocene route](https://user-images.githubusercontent.com/78768991/126246332-25af68b9-a26f-4694-bc34-e569cc9ab28c.png)

## In conclusion 

OK, so we've hit a bit of a sticking point with the propargylation/allylation reactions, but once we've got them to work, we should be able to make good progress as we already have at least one pendant group ready to go. Then there's all the variations of scaffold to try, mostly by simply increasing chain length. For now, we're keeping our fingers crossed that the covid outbreak settles down ASAP and we can get back to work as per usual! 

Remember to follow us on Twitter ([@OpenSourceTB](https://twitter.com/OpenSourceTB)) to keep up to date with the trials and tribulations of open-source drug discovery in a pandemic!

**Comment by mattodd** [comment]
Nice update, thanks. I can imagine that working with some of the diamines with non-aromatic pendants might be quite tricky. What about including something else in the linker region like a naphthalene, or other aromatic. This would rigidify the linker in a way that's probably undesirable, but might make the chemistry easier to explore.

I was also wondering about whether you could change the order of events in the synthesis. You're starting with the alkyl diamine and building it up. Is there any point in building up the separate halves of the molecule you want, as building blocks, and then unifying them with e.g. metathesis or something? i.e. making your red linker region last?

**Comment by MFernflower** [comment]
@mattodd @KlementineJBS Thoughts on the proposed tetrapendant compounds #2 ?
